reduxApp2/dev/js/actions/index.js

```diff
@@ -0,0 +1,6 @@
+export const addUserDetails = (userDetails) => {
```

Comment:
@VARTIKASINGH1234 Because you're using arrow operator, I think we don't here to write return explicitly. Please think about this and if possible convert this into one line statement.

Reply:
removed return.